<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Giải toán</string>
  </property>
  <widget class="QWebEngineView" name="webgiai" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>551</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QWebEngineView" name="webhd" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>311</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_huongdan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>87</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hướng dẫn:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nhap">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>7</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nhập:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_nhap">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_xemtruoc">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>67</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Xem trước:</string>
   </property>
  </widget>
  <widget class="QWebEngineView" name="webxem" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_giaichitiet">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>298</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Giải chi tiết:</string>
   </property>
  </widget>
  <widget class="QWebEngineView" name="webkq" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>551</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_kq">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Kết quả:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_giai">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Giải</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_thoat">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_kt">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>62</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kiểm tra</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_xoa">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>62</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Nhập lại</string>
   </property>
  </widget>
  <widget class="QWebEngineView" name="webdang" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_dang">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_bien">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>164</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Theo biến:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_bien">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>162</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtWebEngineWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_thoat</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>-32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_xoa</sender>
   <signal>clicked()</signal>
   <receiver>lne_nhap</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>683</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
